apps/indexer/lib/indexer/block/fetcher.ex

```diff
@@ -191,7 +229,14 @@ defmodule Indexer.Block.Fetcher do
            beneficiary_params_set
            |> add_gas_payments(transactions_with_receipts)
            |> BlockReward.reduce_uncle_rewards(),
-         address_token_balances = AddressTokenBalances.params_set(%{token_transfers_params: token_transfers}),
+         address_token_balances_from_transfers =
+           AddressTokenBalances.params_set(%{token_transfers_params: token_transfers}),
+         address_token_balances =
```

Comment:
Here all the modified addresses are added to the list for which the new token balances will be checked.